index.html

```diff
@@ -0,0 +1,143 @@
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/EnglishPremierLeague.webp" alt="premier">
+                </div>
+                <div class="title-card">
+                    <h4>premier league</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+
+            <div class="card card-1">
+                <div class="img-card">
+                    <img src="./images/serie.jpg" alt="serie">
+                </div>
+                <div class="title-card">
+                    <h4>serie A</h4>
+                </div>
+                <div class="description-card">
+                    <p>
+                        FIFA adopts temporary employment and registration rules to address several issues in relation to war in Ukraine
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="featured" id="Featured">
+    
+    </section>
+
+    <section class="partner" id="Studio">
+        <h3>Our Partner</h3>
+        <div class="border border-dir"></div>
+        <ul class="Partner-logo">
+          <li><img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/7ecf457666d9fb6c/original/fwc2022_fp_dalian-wanda-co_lbgr.eps?tx=c_fill,x_57,y_28,g_xy_center,q_auto,w_114,h_57" alt="Wanda group" title="Wanda group" class="image_img__JuDY4"></li>
+          <li><img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/7b71c2f3185bddfb/original/fwc2022_fp_cocacola-co_lbgr.eps?tx=c_fill,x_57,y_28,g_xy_center,q_auto,w_114,h_57" alt="Coca Cola" title="Coca Cola" class="image_img__JuDY4"></li>
+          <li>
+            <img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/6d3a4cc8e08eeb0f/webimage-fwc2022_fp_qatar-co_lbgr.png?tx=c_fill,g_auto,q_auto,w_114,h_57" alt="Qatar Airways" title="Qatar Airways" class="image_img__JuDY4">
+          </li>
+          <li>
+            <img decoding="async" loading="lazy" height="71" width="94" src="https://digitalhub.fifa.com/transform/fdae2bc1-3678-4471-955d-20e4870b924f/visa_94x71?io=transform:fill,height:71,width:94" alt="VISA" title="VISA" class="image_img__JuDY4">
+          </li>
+          <li>
+            <img decoding="async" loading="lazy" height="57" width="114" src="https://cloudinary.fifa.com/m/6954f7d3fd116343/webimage-fwc2022_fp_hyundai-kia-co_lbgr.png?tx=c_fill,g_auto,q_auto,w_114,h_57" alt="Hyundai" title="Hyundai" class="image_img__JuDY4">
+          </li>
+        </ul>
+      </section>
```

Comment:
- [ ] Please make sure that your partner section design is similar to the original design because I can see some differences between your design and the original design.  As a developer, you need to pay attention to details, and also it is important to stick to the design as much as possible in order to have a beautiful website and a good user experience :+1: 

#### Your design
![Screenshot from 2022-12-02 21-21-40](https://user-images.githubusercontent.com/6615394/205389449-f7eee695-b9f1-4288-ad04-af4251db80e3.png)

#### The original design
![Screenshot from 2022-12-02 21-25-33](https://user-images.githubusercontent.com/6615394/205389819-89e8582f-cd16-474a-8520-74c11501abab.png)

Reply:
@codecaiine hello what i suppose to change in this section